src/imitation/util/util.py

```diff
@@ -96,8 +96,8 @@ def make_env(i, this_seed):
 
 def init_rl(
     env: Union[gym.Env, VecEnv],
-    model_class: Type[BaseRLModel] = stable_baselines.PPO2,
-    policy_class: Type[BasePolicy] = MlpPolicy,
+    model_class: Type[BaseAlgorithm] = stable_baselines.PPO2,
```

Comment:
What happens when you replace `stable_baselines.PPO2` with `stable_baselines3.PPO`?

Reply:
`init_rl` is just a stub function at this point (I think it might have done more in the past), which we plan on removing. If it doesn't work here, then instantiating PPO should be fine.

Reply:
Do you want to try making this substitution in your draft of the code @qxcv? None of the test scripts I've been working to port actually call this method.

Reply:
Sure, I'll do that.

Reply:
Oops, actually I misspoke: I've already changed it and it works okay (after a few changes to related functions to make them SB3/Torch-compatible).

Reply:
Awesome!